components/setting_item_min.jsx

```diff
@@ -80,6 +80,7 @@ export default class SettingItemMin extends React.PureComponent {
                         className='color--link cursor--pointer style--none'
                         onClick={this.handleUpdateSection}
                         ref={this.getEdit}
+                        aria-labelledby={this.props.section + 'Title ' + this.props.section + 'Edit'}
```

Comment:
I thought AM mention this needs to be translatable?

Reply:
@enahum This is `aria-labelledby`, so it is not itself the label, rather we give it an `id` of an element that it then reads the content of. So it is fine.